### ISSUE TITLE
gradle broken javac options

### ISSUE THREAD

**Issue by aishfenton** [scalacenter/bloop#1493]
Since the v1.4.7 release the Gradle integration is producing a broken set of javac options. Basically, `./gradle bloopInstall` runs fine, but when you run Bloop compile it now has the following exception:

`Unexpected error when compiling my-project_2.11: invalid flag`. 

Poking around a bit I think the problem is that in Bloop 1.4.7+, it started to support annotation processing. This adds an additional `-processorpath` option to the javac options. Which is fine. But there's also an option `-sourcepath` that is always added, that is missing it's path argument (according to the javac man docs, `sourcepath` is supposed to be followed by a path). I think that's probably always been incorrect, but since`-processorpath` was added it's manifesting more obviously in a bug (is my guess).

```json
        "java": {
            "options": [
                "-source",
                "1.8",
                "-target",
                "1.8",
                "-encoding",
                "UTF-8",
                "-h",
                "/User/myUser/myProject/src",
                "-g",
                "-sourcepath",
                "-processorpath",
                "/User/myUser/myProject/src",
                "-s",
                "/Users/...",
                "-XDuseUnsharedTable=true",
                "-implicit:none"
            ]
        }, 
```

**Comment by aishfenton** [comment]
Can confirm that once I remove the `-sourcepath` option manually (i.e. edit the `.bloop/myProject.json` file everything works fine.

**Comment by Arthurm1** [comment]
Hi @aishfenton 

The javacoptions were being corrupted in v1.4.8.

This should be fixed by #1476 and you can use the latest snapshot of the plugin `1.4.8-19-4d9f966b` to try out the fix

**Comment by aishfenton** [comment]
Can confirm, that fixed it for me. Thanks @Arthurm1
